src/Libraries/Microsoft.Extensions.AI.Abstractions/Microsoft.Extensions.AI.Abstractions.csproj

```diff
@@ -2,15 +2,15 @@
 
   <PropertyGroup>
     <RootNamespace>Microsoft.Extensions.AI</RootNamespace>
-    <Description>Abstractions for generative AI.</Description>
+    <Description>Abstractions representing generative AI components.</Description>
     <Workstream>AI</Workstream>
   </PropertyGroup>
 
   <PropertyGroup>
-    <Stage>preview</Stage>
+    <Stage>normal</Stage>
     <EnablePackageValidation>false</EnablePackageValidation>
     <MinCodeCoverage>82</MinCodeCoverage>
-    <MinMutationScore>0</MinMutationScore>
+    <MinMutationScore>85</MinMutationScore>
```

Comment:
Do our tests actually pass with the mutation stuff? We've never looked at the results (or maybe even run it?), to my knowledge.

Is that support still valuable? Seems like it was an interesting experiment, but should the repo keep it around moving forward?

Reply:
Mutation tests existed in the R9 codebase, but we couldn't get those working in satisfactory manner here; so this is a rudiment waiting a removal.